### ISSUE TITLE
Reenter credential with SSO blank page

### ISSUE THREAD

**Issue by lucadistefano** [microsoft/AzureStorageExplorer#546]
**Storage Explorer Version**: 1.3.1
**Platform/OS Version**: macos 10.13.6
**Architecture**: x64
**Regression From**: 
maybe #259 

#### Steps to Reproduce: ####
1. start storage explorer
2. click on 'Reenter credentials'

#### Expected Experience: ####
in sign up popup have field for enter credentials and button to confirm

#### Actual Experience: ####
the sign in popup is blank

![image](https://user-images.githubusercontent.com/1080723/43723003-ec6bb8fc-9996-11e8-9599-e148ca7fa3ae.png)

In a earlier version this bug was fixed and in the release notes was suggested to remove app preferences. I performed the suggestion and it has worked... up to next credentials expiration

**Comment by MRayermannMSFT** [comment]
Hello to anyone watching this issue! In 1.7.0 we're introducing the ability to optionally sign-in via device code flow, or as you may be more familiar with, copying a code from Storage Explorer into a page in your browser, and then signing in via your browser.

We're currently previewing that feature in our insiders channel. If y'all are intereseted in trying it early, simply go to Help -> Opt-in To Insider Builds.
![image](https://user-images.githubusercontent.com/21112697/52584072-04563f80-2de6-11e9-90c1-094e4bbcd0b4.png)
Then, go to Help -> Check for updates, wait for 1.7.0 to appear, and install it. Then once you install it, you can enable device code flow by going to Experimental -> Use Device Code Flow Sign In. 
![image](https://user-images.githubusercontent.com/21112697/52584173-3e274600-2de6-11e9-85af-6ed3cf5ed006.png)

And then let us know if things work out for y'all! This issue of blank sign in dialog is one we've been fighting for the longest time, and we're hoping this method of sign-in will unblock y'all. 😄

**Comment by MRayermannMSFT** [comment]
Closing. If anyone is still having issues, we now recommend trying out MSAL. MSAL is a newer auth library we'll be switching to in the future. Today though, you can opt into it by going to Settings -> Application -> Sign in -> Use Microsoft Authentication Library (MSAL)

Please open a new issue if you have any problems with MSAL.
